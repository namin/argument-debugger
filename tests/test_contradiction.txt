Life is good. Life is bad.
