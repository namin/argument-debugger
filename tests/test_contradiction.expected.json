{
  "must_find": [
    "contradiction"
  ],
  "may_find": [],
  "must_not_find": []
}